src/Operation/Pluck.php

```diff
@@ -19,6 +19,7 @@
 use function in_array;
 use function is_array;
 use function is_object;
+use function is_scalar;
```

Comment:
hmm did you add this manually or was it added by php-cs-fixer or something automatically? 🤔

Reply:
php-cs-fixer :)

Reply:
cool, I guess it's not set up to import _all native functions_ though, because there are others like `array_shift` and `property_exists` in this file that are not imported

Reply:
It has been added in here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6277

Reply:
I made a commit in master with these CS changes, so they don't pollute this branch.

Reply:
ah I see, I've typically used that rule with the `all` option rather than only the compiler optimized ones just for consistency, which is why I was surprised 😄